sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -158,6 +180,11 @@ def _populate_metric_results(self, response):
                                  attempted=attempted,
                                  committed=committed))
 
+  def _populate_metric_results(self, response):
+    """Take a list of metrics, and convert it to a list of MetricResult."""
+    result = []
+    self._populate_metrics(response, result, user_metrics=True)
```

Comment:
Do we ever want to call these separately?

Reply:
My intention here was the make the query() code only return user metrics, and make all metrics accessible when calling all_metrics(). After a discussion and design on the dev list, we can consider adding the system metrics to the regular query() call.

Fixing this up to reflect this, I can see I forgot to make query() only return user metrics.